src/main/java/frc/robot/CommandSequences.java

```diff
@@ -52,24 +66,59 @@ public Command moveToAmp() {
-        return led.toPreset().andThen(armRotate.moveTo(ArmPresets.SCORE_AMP), led.atPreset(), led.shooterSpinning(),
-                shooter.setSpeed(Speeds.SLOW_SPEED), led.shooterAtSpeed(), intake.feedShooter(),
-                shooter.setSpeed(Speeds.OFF), led.colorAlliance());
+    public Command stow() {
+        return led.toPreset().andThen(armRotate.moveTo(ArmPresets.STOW), led.atPreset());
     }
 
     public Command moveToSpeaker() {
         return led.toPreset().andThen(armRotate.moveTo(ArmPresets.SCORE_SPEAKER_SUBWOOFER), led.atPreset());
     }
 
-    public Command scoreSpeaker() {
-        return led.toPreset().andThen(armRotate.moveTo(ArmPresets.SCORE_SPEAKER_SUBWOOFER), led.atPreset(),
-                led.shooterSpinning(), shooter.setSpeed(Speeds.THREE_QUARTER_SPEED), led.shooterAtSpeed(),
+    public Command scoreAmp() {
+        return this.armRotatePreset(ArmPresets.SCORE_AMP).withTimeout(0.75).andThen(this.shooterSpeed(Speeds.AMP_SPEED),
                 intake.feedShooter(),
                 shooter.setSpeed(Speeds.OFF), led.colorAlliance());
     }
 
-    public Command stow() {
-        return led.toPreset().andThen(armRotate.moveTo(ArmPresets.STOW), led.atPreset());
+    public Command scoreSpeakerCharge(ButtonXboxController controller2, ButtonXboxController controller1) {
+        return this.shooterSpeed(Speeds.SUBWOOFER_SHOT).alongWith(
+                this.armRotatePreset(ArmPresets.SCORE_SPEAKER_SUBWOOFER)).andThen(
+                        setRumble(controller1, 1), setRumble(controller2, 1));
+    }
+
+    public Command scoreSpeakerRelease(ButtonXboxController controller2, ButtonXboxController controller1) {
+        return intake.feedShooter().andThen(shooter.setSpeed(Speeds.OFF), setRumble(controller1, 0),
+                setRumble(controller2, 0));
+    }
+
+    public Command scoreSpeakerAuto() {
+        return this.shooterSpeed(Speeds.SUBWOOFER_SHOT).alongWith(
+                this.armRotatePreset(ArmPresets.SCORE_SPEAKER_SUBWOOFER)).andThen(
+                        intake.feedShooter(),
+                        led.colorAlliance());
+    }
+
+    public Command podiumShot() {
+        return this.shooterSpeed(Speeds.PODIUM_SHOT)
+                .alongWith(this.armRotatePreset(ArmPresets.SCORE_SPEAKER_PODIUM))
+                .andThen(intake.feedShooter(), shooterSpeed(Speeds.OFF));
+    }
+
+    public Command setRumble(ButtonXboxController controller, int rumbleAmount) {
+        return runOnce(() -> {
+            controller.getHID().setRumble(RumbleType.kBothRumble, rumbleAmount);
+        });
+    }
+
+    public Command test(ButtonXboxController controller) {
+        return shooter.setSpeed(Speeds.HALF_SPEED).andThen(() -> {
+            this.setRumble(controller, 1);
+        });
+    }
+
+    public Command rumbleAndShoooterOff(ButtonXboxController controller) {
+        return shooter.setSpeed(Speeds.HALF_SPEED).andThen(() -> {
+            this.setRumble(controller, 0);
```

Comment:
Same here, `setRumble` returns a command so you don't need to wrap it in a lambda